packages/blade/src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Button from './Button';
+
+const ButtonMeta: ComponentMeta<typeof Button> = {
+  title: 'Components/Button',
+  component: Button,
+  argTypes: {
+    onPress: { action: 'pressed the button' },
+  },
+  args: {
+    text: 'Hello world',
+  },
+};
+export default ButtonMeta;
+
+const Basic: ComponentStory<typeof Button> = (args) => <Button {...args} />;
```

Comment:
What is this for?

Reply:
a temp story will remove it. wanted to check if the setup is working fine